examples/custom_secrets/no-store.us-east-1.tfvars

```diff
@@ -0,0 +1,15 @@
+region = "us-east-1"
+
+namespace = "eg"
+
+environment = "ue1"
+
+stage = "test"
+
+name = "self-signed-cert-ssm-disabled"
+
+secret_path_format = "/ssm-test/%s.%s"
+
+secrets_store_enabled = false
+
+secrets_store_type = "SSM"
```

Comment:
if we set `SSM` as default, we won't need to specify it here when `secrets_store_enabled = false`